crates/typst-pdf/src/font.rs

```diff
@@ -146,7 +146,7 @@ pub(crate) fn write_fonts(ctx: &mut PdfContext) {
 
         // Write the /ToUnicode character map, which maps glyph ids back to
         // unicode codepoints to enable copying out of the PDF.
-        let cmap = create_cmap(ttf, glyph_set);
+        let cmap = create_cmap(ttf, glyph_set, font);
```

Comment:
Passing `ttf` and `font` is a bit redundant as we can cheaply reacquire `ttf` from `font`. Could you adjust the signature to `create_cmap(font, glyph_set)`?

Reply:
Sure! I didn't know `Font` is an `Arc`.